requirements.txt

```diff
@@ -2,6 +2,7 @@ attrs
 boltons
 dask
 doct
+glueviz
```

Comment:
probably better to put this in `test-requirements` rather than `requirements`, glue is going to pull in some pretty heavy dependencies which we may not want in all cases.